content/blogposts/2023/sourcegraph-5.0.md

```diff
@@ -0,0 +1,251 @@
+
+<Badge link="/batch-changes" text="Batch Changes" color="blue" size="small" />
+#### Limit access to batch changes to designated users
+
+While batch changes enable efficient bulk creation of changesets, some customers require tighter controls to meet the security and compliance standards of their industry. To meet these needs, we're introducing a role-based access control (RBAC) system to Sourcegraph to enable more granular controls for who can access batch changes.
+Site admins can limit access to approved users from admin settings under “Site admin” > “Users & auth” > “Roles” by:
+
+- Creating custom user roles with Batch Changes permissions
+- Revoking default role permissions
+- Assigning designated users to Batch Changes-enabled roles
+<br/>
+
+While the RBAC system is limited to batch changes for now, the system will soon be expanded to cover additional areas of Sourcegraph as well.
+
+<br/>
+<Video 
+  source={{
+    webm: 'blog/5.0/batch-changes-rbac',
+    mp4: 'blog/5.0/batch-changes-rbac'
+  }}
+  loop={true}
+  title="RBAC for Batch Changes"
+/>
+<br/>
+
+<a href="https://docs.sourcegraph.com/admin/access_control" className="not-italic flex items-center mb-sm">Docs<OpenInNewIcon className="ml-xxs" size={18} /></a>
+<br />
+
+<Badge text="Code Insights" link="/code-insights" color="green" size="small" />
+#### Improved Code Insights support for instances with a large number of repositories
+
+Code insights provide precise answers about the trends and composition of your codebase, tranforming it into a queryable database.  Previously, users had to create insights over either a few named repositories or all of their repositories, and running insights over all repositories could often take an impractical amount of time to process. Now, with the new repository selection, users can target their insights to the exact scope of repositories that is relevant to them. Additionally, administrators can now monitor the processing of insights and, when necessary, reprioritize or retry them from within the site admin section.
+
+<a href="https://docs.sourcegraph.com/code_insights" className="not-italic flex items-center mb-sm">Docs<OpenInNewIcon className="ml-xxs" size={18} /></a>
+<br />
+
+<Badge link="https://docs.sourcegraph.com/admin/external_service" text="Code Hosts" color="violet" size="small" />
+#### Impoved Gerrit support with user permissions
+
+Gerrit connections now have their own dedicated code host connection option as opposed to the “Generic Git Host” connection that had to be used before. Along with this, Gerrit connections can now also enable authorization, which will mark all repositories from that connection as private and require users to authenticate using Gerrit credentials and verify that they have permission to view that repository on Gerrit if they want to be able to view it on Sourcegraph.
```

Comment:
I'm not sure about

> which will mark all repositories from that connection as private

@pjlast @kopancek -- is that the correct way to phrase it?

Reply:
Yeah, that sounds good to me!

Reply:
```suggestion
Gerrit connections now have their own dedicated code host connection option as opposed to the “Generic Git Host” connection that had to be used before. Gerrit connections can also be configured to require user authentication, in which case users will only be able to view repositories by providing their own Gerrit credentials.
```